apps/aevm/src/aevm_chain.erl

```diff
@@ -0,0 +1,140 @@
+%%%=============================================================================
```

Comment:
I would like this to be in aecore in order to not have dependencies in two directions.
The thought is that you should be able to take the aevm app and build it without the rest.
This would make @ThomasArts system graph look nicer (or at least not make it look more complex).

Reply:
Yes, I agree.